ui/lib/account-link.js

```diff
@@ -18,6 +18,7 @@ module.exports = function (address, network) {
       link = `https://kovan.etherscan.io/address/${address}`
       break
     default:
+      // FIXME
```

Comment:
Does [GasTracker.io](https://gastracker.io/) work for this purpose?

Reply:
yes, would be `https://gastracker.io/addr/${address}`